build.sbt

```diff
@@ -17,7 +17,7 @@ ThisBuild / developers := List(
     id    = "SP",
```

Comment:
Should this be your details? Have we got a generic TDR account that could be used instead?

Reply:
I've put the Jenkins GitHub user in there now.